src/pipx/commands/inject.py

```diff
@@ -98,10 +105,19 @@ def inject(
     suffix: bool = False,
 ) -> ExitCode:
     """Returns pipx exit code."""
+    # Combined list of packages
+    packages = list(package_specs)
+    for filename in requirement_files:
+        packages.extend(parse_requirements(filename))
```

Comment:
We probably should deduplicate packages here.

Reply:
That would be ideal, but we don't do that with `package_specs` and pip will just handle it, I expect. So IMO that's nice-to-have, rather than necessary.